ios/vendored/unversioned/react-native-reanimated/ios/native/NativeProxy.mm

```diff
@@ -180,13 +176,12 @@ static id convertJSIValueToObjCObject(jsi::Runtime &runtime, const jsi::Value &v
     return val;
   };
 
-#if __has_include(<reacthermes/HermesExecutorFactory.h>)
-  std::shared_ptr<jsi::Runtime> animatedRuntime = facebook::hermes::makeHermesRuntime();
-#elif __has_include(<hermes/hermes.h>)
-  std::shared_ptr<jsi::Runtime> animatedRuntime = facebook::hermes::makeHermesRuntime();
-#else
-  std::shared_ptr<jsi::Runtime> animatedRuntime = facebook::jsc::makeJSCRuntime();
-#endif
+  std::shared_ptr<jsi::Runtime> animatedRuntime;
+  if (useHermes) {
```

Comment:
What if we always use Hermes for reanimated runtime? I guess it would simplify versioning 🤔